jenkins/jdbc-stage-maven-release.jenkinsfile

```diff
@@ -39,7 +39,7 @@ pipeline {
         )
     }
     environment {
-        ARTIFACT_PATH = "$WORKSPACE/build/repository/org/opensearch/client/sql/$VERSION"
+        ARTIFACT_PATH = "$WORKSPACE/build/repository/org/opensearch/driver/opensearch-sql-jdbc/$VERSION"
```

Comment:
We used to put the jdbc in this location during ODFE.
Probably we want to keep the same subfolder structure?

https://repo1.maven.org/maven2/com/amazon/opendistroforelasticsearch/client/opendistro-sql-jdbc/

Thanks.

Reply:
It is the path where gradle places compiled `jar`:
```
$ tree build | grep -v class
build
....
├── repository
│   └── org
│       └── opensearch
│           └── driver
│               └── opensearch-sql-jdbc
│                   ├── 2.0.0.0
│                   │   ├── opensearch-sql-jdbc-2.0.0.0.jar
....
```

In general, there is the same structure, but we renamed 
* `com.amazon` to `org.opensearch`
* `opendistroforelasticsearch` and `opendistro` to `opensearch`
* `client` to `driver`

Reply:
Should I revert this?